Source/Softswitch/inc/poets_msg.h

```diff
@@ -17,6 +17,7 @@
 #define P_DEVICE_OS 0
 #define P_THREAD_OS (LOG_DEVICES_PER_THREAD)
 #define P_CORE_OS (LOG_THREADS_PER_CORE+P_THREAD_OS)
+#define P_MAILBOX_OS (TinselLogCoresPerMailbox+P_CORE_OS)
```

Comment:
I vaguely remember @heliosfa talking to me about adding this. I reckon he should chime in here (instead of me).

And yes, this was my mistake. Sorry! :cactus: